src/models/Projections.ts

```diff
@@ -122,8 +148,9 @@ export class Projections {
     if (lowCount === 3) {
       // if all the factors are low, level is low
       level = Level.LOW;
-    } else if (unKnownCount === 3) {
+    } else if (unKnownCount === 3 || (mediumCount === 0 && highcount === 0)) {
       // if all the levels are unkown, status is unkwonw
+      // no reds and no yellows, but at least one unknown
       level = Level.UNKNOWN;
```

Comment:
I think you could simplify this a little to:

```
} else if (unKnownCount + lowCount === 3) {
  // no reds and no yellows, but at least one unknown.
  level = Level.UNKNOWN;
}
```

and then you can drop `highCount` if you want.
}
```